R-packages/covidcast/R/wrangle.R

```diff
@@ -281,7 +281,7 @@ covidcast_longer = function(x) {
 #' @rdname covidcast_longer
 #' @export
 covidcast_wider = function(x) {
-  if (!("covidcast_signal_long" %in% class(x))) {
+  if (!inherits(x, "covidcast_signal_long")) {
```

Comment:
Good call, and can you please change the other two appearances of this logic: `if ("covidcast_signal" %in% class(x))`, in wrangle.R (I found them just now with grep).